debezium-storage/debezium-storage-configmap/src/main/java/io/debezium/storage/configmap/ConfigMapFormatter.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.storage.configmap;
+
+import java.nio.ByteBuffer;
+import java.util.Arrays;
+import java.util.Base64;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import io.debezium.DebeziumException;
+
+/**
+ * This class manage the format of key used for Kubernetes config map
+ * <p>
+ * The keys are converted to a valid config:
+ * alphanumeric characters, '-', '_' or '.' (e.g. 'key.name',  or 'KEY_NAME',  or 'key-name',
+ * regex used for validation is '[-._a-zA-Z0-9]+').
+ * <p>
+ * The values are just base64 encoded
+ *
+ * @author Mario Fiore Vitale
+ */
+public class ConfigMapFormatter {
```

Comment:
We need a better docs for this -- or better a test class for format specifically